packages/client/modules/teamDashboard/components/EditTeamName/EditableTeamName.tsx

```diff
@@ -29,7 +29,7 @@ const EditableTeamName = (props: Props) => {
         teamId: id
         teamName: name
         organization {
-          teams {
+          viewerTeams {
```

Comment:
-1 This is used to check for duplicate names in l59, so it should be `allTeams` here.